### PR TITLE
IA Migration day: Update Apache redirects, URLs, and site footer

### PR DESCRIPTION
**Pull request by wpears**
Addresses:
- https://github.local/CFPB/super-regular/issues/150
- https://github.local/CFPB/super-regular/issues/149
- https://github.local/CFPB/super-regular/issues/167
- https://github.local/CFPB/super-regular/issues/157

1. Add all necessary Apache redirects for the IA Refresh migration
2. Update any existing redirects that are affected by the IA Refresh migration, to ensure we aren't creating redirect chains
3. Update URLs that are defined in Django's `urls.py` to move them to their new URLs.
4. Move some redirects that are defined in Django to Apache's `redirects.conf` instead, to start to reduce the number of places we're managing redirects.
5. Update the global footer to reflect the new IA

We'll be merging and deploying this PR as part of the IA Refresh migration on Wednesday, September 30.

### CODE REVIEW

File: cfgov/apache/conf.d/redirects.conf [hunk 1]
```diff
@@ -52,24 +52,24 @@ equal-credit-opportunity-act-valuation-rule/|\
 rural-and-underserved-counties-list/|\
 secure-fair-enforcement-for-mortgage-licensing-act/.*|\
 mortserv/.*\
-) /policy-compliance/guidance/mortgage-resources/$1
+) /policy-compliance/guidance/compliance-resources/mortgage-resources/$1
```
Comment (chosak) [inline cfgov/apache/conf.d/redirects.conf:55]:
```suggestion
) /compliance/compliance-resources/mortgage-resources/$1
```

File: cfgov/apache/conf.d/redirects.conf [hunk 2]
```diff
@@ -52,24 +52,24 @@ equal-credit-opportunity-act-valuation-rule/|\
...
-Redirect permanent /policy-compliance/guidance/tila-higher-priced-mortgage-loan-escrow-rule/ /policy-compliance/guidance/mortgage-resources/higher-priced-mortgage-loan-escrow-rule/
-Redirect permanent /policy-compliance/guidance/payday-lending-rule/ /policy-compliance/guidance/consumer-lending-resources/payday-lending-rule/
-Redirect permanent /policy-compliance/guidance/prepaid-rule/ /policy-compliance/guidance/consumer-cards-resources/prepaid-cards/
-Redirect permanent /policy-compliance/guidance/truth-lending-annual-threshold-adjustments/ /policy-compliance/guidance/consumer-cards-resources/truth-lending-annual-threshold-adjustments/
-Redirect permanent /policy-compliance/guidance/remittance-transfer-rule/ /policy-compliance/guidance/deposit-accounts-resources/remittance-transfer-rule/
-Redirect permanent /policy-compliance/guidance/gramm-leach-bliley-act/ /policy-compliance/guidance/other-applicable-requirements/privacy-notices/
+) /policy-compliance/guidance/compliance-resources/other-applicable-requirements/$1
```
Comment (chosak) [inline cfgov/apache/conf.d/redirects.conf:60]:
```suggestion
) /compliance/compliance-resources/other-applicable-requirements/$1
```

File: cfgov/apache/conf.d/redirects.conf [hunk 3]
```diff
@@ -52,24 +52,24 @@ equal-credit-opportunity-act-valuation-rule/|\
...
+
+Redirect permanent /policy-compliance/guidance/hmda-implementation/ /policy-compliance/guidance/compliance-resources/mortgage-resources/hmda-reporting-requirements/
+Redirect permanent /policy-compliance/guidance/home-ownership-and-equity-protection-act-rule/ /policy-compliance/guidance/compliance-resources/mortgage-resources/high-cost-mortgages/
+Redirect permanent /policy-compliance/guidance/loan-originator-rule/ /policy-compliance/guidance/compliance-resources/mortgage-resources/loan-origination-rule/
+Redirect permanent /policy-compliance/guidance/tila-respa-disclosure-rule/ /policy-compliance/guidance/compliance-resources/mortgage-resources/tila-respa-integrated-disclosures/
+Redirect permanent /policy-compliance/guidance/tila-higher-priced-mortgage-loans-appraisal-rule/ /policy-compliance/guidance/compliance-resources/mortgage-resources/higher-priced-mortgage-loans-appraisal-rule/
+Redirect permanent /policy-compliance/guidance/tila-higher-priced-mortgage-loan-escrow-rule/ /policy-compliance/guidance/compliance-resources/mortgage-resources/higher-priced-mortgage-loan-escrow-rule/
```
Comment (chosak) [inline cfgov/apache/conf.d/redirects.conf:67]:
```suggestion
Redirect permanent /policy-compliance/guidance/hmda-implementation/ /compliance/compliance-resources/mortgage-resources/hmda-reporting-requirements/
Redirect permanent /policy-compliance/guidance/home-ownership-and-equity-protection-act-rule/ /compliance/compliance-resources/mortgage-resources/high-cost-mortgages/
Redirect permanent /policy-compliance/guidance/loan-originator-rule/ /compliance/compliance-resources/mortgage-resources/loan-origination-rule/
Redirect permanent /policy-compliance/guidance/tila-respa-disclosure-rule/ /compliance/compliance-resources/mortgage-resources/tila-respa-integrated-disclosures/
Redirect permanent /policy-compliance/guidance/tila-higher-priced-mortgage-loans-appraisal-rule/ /compliance/compliance-resources/mortgage-resources/higher-priced-mortgage-loans-appraisal-rule/
Redirect permanent /policy-compliance/guidance/tila-higher-priced-mortgage-loan-escrow-rule/ /compliance/compliance-resources/mortgage-resources/higher-priced-mortgage-loan-escrow-rule/
```

File: cfgov/apache/conf.d/redirects.conf [hunk 4]
```diff
@@ -52,24 +52,24 @@ equal-credit-opportunity-act-valuation-rule/|\
...
+Redirect permanent /policy-compliance/guidance/payday-lending-rule/ /policy-compliance/guidance/compliance-resources/consumer-lending-resources/payday-lending-rule/
+Redirect permanent /policy-compliance/guidance/prepaid-rule/ /policy-compliance/guidance/compliance-resources/consumer-cards-resources/prepaid-cards/
+Redirect permanent /policy-compliance/guidance/truth-lending-annual-threshold-adjustments/ /policy-compliance/guidance/compliance-resources/consumer-cards-resources/truth-lending-annual-threshold-adjustments/
+Redirect permanent /policy-compliance/guidance/remittance-transfer-rule/ /policy-compliance/guidance/compliance-resources/deposit-accounts-resources/remittance-transfer-rule/
+Redirect permanent /policy-compliance/guidance/gramm-leach-bliley-act/ /policy-compliance/guidance/compliance-resources/other-applicable-requirements/privacy-notices/
```
Comment (chosak) [inline cfgov/apache/conf.d/redirects.conf:72]:
```suggestion
Redirect permanent /policy-compliance/guidance/payday-lending-rule/ /compliance/compliance-resources/consumer-lending-resources/payday-lending-rule/
Redirect permanent /policy-compliance/guidance/prepaid-rule/ /compliance/compliance-resources/consumer-cards-resources/prepaid-cards/
Redirect permanent /policy-compliance/guidance/truth-lending-annual-threshold-adjustments/ /compliance/compliance-resources/consumer-cards-resources/truth-lending-annual-threshold-adjustments/
Redirect permanent /policy-compliance/guidance/remittance-transfer-rule/ /compliance/compliance-resources/deposit-accounts-resources/remittance-transfer-rule/
Redirect permanent /policy-compliance/guidance/gramm-leach-bliley-act/ /compliance/compliance-resources/other-applicable-requirements/privacy-notices/
```

File: cfgov/apache/conf.d/redirects.conf [hunk 5]
```diff
@@ -993,7 +990,7 @@ Redirect /assets https://files.consumerfinance.gov/a/assets
 
 # Redirects, requested by Rachael Keeler in July 2019
 # https://GHE/CFGOV/platform/issues/3484
-RedirectMatch permanent (?i)^/about-us/advisory-groups/(.*) /about-us/advisory-committees/$1
+RedirectMatch permanent (?i)^/about-us/advisory-groups/(.*) /policy-compliance/rulemaking/advisory-committees/$1
```
Comment (chosak) [inline cfgov/apache/conf.d/redirects.conf:993]:
```suggestion
RedirectMatch permanent (?i)^/about-us/advisory-groups/(.*) /rules-policy/advisory-committees/$1
```

**Review by chosak — COMMENTED** [review]
@wpears I made what I think are corrections based on comparing your changes with the current spreadsheet (I still haven't _added_ any new redirects, just checking your proposals).

I also found a few redirect chains that you might have missed - I'll add in another commit.

**Comment by wpears** [comment]
@chosak Thanks for those additions, included them in a rebase then found quite a few more from the updated spreadsheet (added in e194b1c4)

**Review by chosak — APPROVED** [review]
These look good to me.

File: cfgov/apache/conf.d/redirects.conf [hunk 6]
```diff
@@ -983,6 +991,9 @@ RedirectMatch permanent (?i)^/practitioner-resources/servicemembers/planning/?$
 RedirectMatch permanent (?i)^/practitioner-resources/servicemembers/planning/creativesavingsstrategies/?$ /consumer-tools/military-financial-lifecycle/
 RedirectMatch permanent (?i)^/practitioner-resources/servicemembers/protecting/?$ /consumer-tools/military-financial-lifecycle/
 
+# 9/30/20 IA work, practitoiner-resources line moved here to allow the above redirects to maintain effect
```
Comment (chosak) [inline cfgov/apache/conf.d/redirects.conf:994]:
```suggestion
# 9/30/20 IA work, practitioner-resources line moved here to allow the above redirects to maintain effect
```

**Pull request merged**